RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitWorkerSimple.cc

```diff
@@ -99,7 +99,7 @@ bool HGCalRecHitWorkerSimple::run(const edm::Event & evt, const HGCUncalibratedR
         hid = detid;
         thickness = ddds_[hid.subdetId() - 3]->waferTypeL(hid.wafer());
         sigmaNoiseGeV = 1e-3 * weights_[layer] * rcorr_[thickness]
-                    * hgcHEF_noise_fC_[thickness - 1] / hgcHEF_fCPerMIP_[thickness - 1];
+                    * hgcEE_noise_fC_[thickness - 1] / hgcEE_fCPerMIP_[thickness - 1];
```

Comment:
@perrotta yes, this is a bugfix for a typo and I confirm that the two cases should be treated the same.